doc/api/v0.yaml

```diff
@@ -559,3 +581,16 @@ components:
           items:
               type: integer
               format: i64
+
+    VoterGroup:
+      properties:
+        id: integer
```

Comment:
I think a string could offer more flexibility and possibly be more descriptive if we want to go that way